test/integration/ies.go

```diff
@@ -36,12 +35,11 @@ var uplinkPDR = ie.NewCreatePDR(
 	ie.NewQERID(4),
 )
 
-
 // TODO: use builder pattern to create PDR IE
 func NewUplinkPDR(method IEMethod, id uint16, teid uint32, n3address string,
```

Comment:
IMO these methods should be implemented using Builder pattern and moved to the `pfcpsim` package